File: FrontEnd/adminuser/src/Interfaces/usuarios.ts
import { Cargos } from "./cargos";
import { Departamentos } from "./departamentos";

export interface Usuarios {
    id:number,
    usuario:string,
    correo:string,
    primerNombre:string,
    segundoNombre:string,
    primerApellido:string,
    segundoApellido:string,
    idDepartamento:Departamentos,
    idCargo:Cargos,
    
}

File: BackEnd/AdminUsers/AdminUsers/Models/Usuarios.cs
using System.ComponentModel.DataAnnotations;

namespace AdminUsers.Models
{
    public class Usuario
    {
        
        public int id { get; set; }
        public string usuario { get; set; }
        public string correo { get; set; }
        public string primerNombre { get; set; }
        public string segundoNombre { get; set; }
        public string primerApellido { get; set; }
        public string segundoApellido { get; set; }
        public Departamento? idDepartamento { get; set; }
        public Cargo? idCargo { get; set; }
    }
}
